modAionImpl/src/org/aion/zero/impl/db/AionRepositoryImpl.java

```diff
@@ -213,37 +219,24 @@ public void updateBatch(
                 }
             }
 
-            if (LOG.isTraceEnabled()) {
-                LOG.trace("updated: detailsCache.size: {}", detailsCache.size());
+            for (Map.Entry<AionAddress, TransformedCodeInfo> entry : transformedCodeCache.entrySet()) {
+                transformedCodeSource.put(entry.getKey().toByteArray(), entry.getValue());
```

Comment:
I think this should be calling `setTransformedCode` otherwise it might overwrite old entries